### PR TITLE
New and updated TissueSampleTypes

### PR DESCRIPTION
**Pull request by UlrikeS91**
#55

### CODE REVIEW

File: instances/tissueSampleType/biopsySample.jsonld [hunk 1]
```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/tissueSampleType/biopsySample",
   "@type": "https://openminds.ebrains.eu/controlledTerms/TissueSampleType",
-  "definition": null,
+  "definition": "A sample of tissue or cell that was excised from a living or deceased multicellular organism body.",
```
Comment (lzehl) [inline instances/tissueSampleType/biopsySample.jsonld]: is that the definition from interlex? "or cell" seems weird to me... I would suggest:
"Typically a very small sample of tissue that was excised from a living or deceased multicellular organism body."

File: instances/tissueSampleType/hemisphere.jsonld [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/tissueSampleType/hemisphere",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/TissueSampleType",
+  "definition": "A hemisphere that was excised from a bilateral organ tissue sample (e.g., a brain) from a living or deceased multicellular organism body.",
```
Comment (lzehl) [inline instances/tissueSampleType/hemisphere.jsonld]: Suggestion: "One of the symmetric halfs of an excised bilateral organ tissue sample ..."

File: instances/tissueSampleType/homogeneousCellPopulation.jsonld [hunk 3]
```diff
@@ -2,13 +2,13 @@
...
-  "definition": "Cells taken from a living organism and grown under controlled conditions (in culture).",
+  "definition": "A sample of multiple cells/a population of cells that are of the same cell type.",
   "description": null,
   "interlexIdentifier": null,
   "knowledgeSpaceLink": null,
-  "name": "cell culture",
-  "preferredOntologyIdentifier": "http://purl.obolibrary.org/obo/BTO_0000214",
+  "name": "homogeneous cell population",
+  "preferredOntologyIdentifier": null,
   "synonym": null
```
Comment (lzehl) [inline instances/tissueSampleType/homogeneousCellPopulation.jsonld:13]: maybe here you should mention: cell line ?

File: instances/tissueSampleType/nerve.jsonld [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/tissueSampleType/nerve",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/TissueSampleType",
+  "definition": "A nerve sample (i.e. a whole nerve or a part of a nerve) from a living or deceased multicellular organism body.",
```
Comment (lzehl) [inline instances/tissueSampleType/nerve.jsonld:7]: suggestion "... from a neuron of a ..."

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/tissueSampleType/biopsySample.jsonld (hunk 1 above)
Comment (UlrikeS91) [reply instances/tissueSampleType/biopsySample.jsonld]: Yeah, it's from InterLex (modifed though). This is what they state: "A specimen collection that obtains a sample of tissue or cell from a living multicellular organism body for diagnostic purposes by means intended to be minimally invasive."

File: instances/tissueSampleType/homogeneousCellPopulation.jsonld (hunk 3 above)
Comment (UlrikeS91) [reply instances/tissueSampleType/homogeneousCellPopulation.jsonld:13]: I wasn't sure about that. With broadening the name to homogeneous cell population, I don't think "cell line" would be a synonym anymore, but rather an example of such a population. What do you think?

File: instances/tissueSampleType/nerve.jsonld (hunk 4 above)
Comment (UlrikeS91) [reply instances/tissueSampleType/nerve.jsonld:7]: I think we have different definitions for a nerve...

Merriam-Webster defines a nerve as "any of the filamentous bands of nervous tissue that connect parts of the nervous system with the other organs, conduct nerve impulses, and are made up of axons and dendrites together with protective and supportive structures". 

To me that's more than just a neuron, isn't it?

File: instances/tissueSampleType/biopsySample.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/tissueSampleType/biopsySample.jsonld]: Then use a mix:
 "Typically a very small sample of tissue that was excised from a living or deceased multicellular organism body for diagnostic purposes by means intended to be minimally invasive/destructive."

"destructive I've added for the deceased organisms... not sure if invasive makes sense there

File: instances/tissueSampleType/homogeneousCellPopulation.jsonld (hunk 3 above)
Comment (lzehl) [reply instances/tissueSampleType/homogeneousCellPopulation.jsonld:13]: not sure. leave it out

File: instances/tissueSampleType/nerve.jsonld (hunk 4 above)
Comment (lzehl) [reply instances/tissueSampleType/nerve.jsonld:7]: no ... I had the same definition in mind (should have written neurons though)... but leave it as it is

**Review by lzehl — APPROVED** [review]

**Pull request merged**